### PR TITLE
Connected profile bio stats challenge and social tabs to server

### PR DESCRIPTION
**Pull request by Paul-Vasylenko**

### CODE REVIEW

**Comment by Paul-Vasylenko** [comment]
[User profile page: Connect to server (MVP)](https://trello.com/c/wbsVi935/162-user-profile-page-connect-to-server-mvp)

**Comment by Paul-Vasylenko** [comment]
[[Profile Page] connect to server Social](https://trello.com/c/1iCgtVVw/165-profile-page-connect-to-server-social)

File: client/src/containers/profile/index.tsx [hunk 1]
```diff
@@ -36,6 +34,131 @@ export const Profile = (props: RouteComponentProps) => {
 		);
 	}, [username]);
 
+	const mockPointsProps = {
+		rank: userData ? (userData.rank ? userData.rank : 9) : 9,
```
Comment (svitlua) [inline client/src/containers/profile/index.tsx:38]: here you can simply write:
rank: userData?.rank ?? 9

and the same for honor and completedChallenged

File: client/src/containers/profile/logic/saga.ts [hunk 2]
```diff
@@ -1,14 +1,126 @@
...
+				return user;
+			}),
+		);
+		let followingsSocial: any[] = yield all(
+			followings.map((following: any) => {
+				const user = call(getUserById, following.user);
+				return user;
+			}),
+		);
+		let comminitySocial: any[] = yield all(
```
Comment (svitlua) [inline client/src/containers/profile/logic/saga.ts:97]: typo here
should be comm**U**nitySocial

**Review by svitlua — CHANGES_REQUESTED** [review]
Please, simplify ternary using ?? operator

File: client/src/containers/profile/index.tsx (hunk 1 above)
Comment (Paul-Vasylenko) [reply client/src/containers/profile/index.tsx:38]: nice advice, thanks

**Review by svitlua — APPROVED** [review]

File: client/src/containers/profile/index.tsx [hunk 3]
```diff
@@ -36,6 +34,129 @@ export const Profile = (props: RouteComponentProps) => {
 		);
 	}, [username]);
 
+	const mockPointsProps = {
+		rank: userData?.rank ?? 9,
```
Comment (svitlua) [inline client/src/containers/profile/index.tsx:38]: here, instead of 9 would be good to use come constant like DEFAULT_RANK

**Pull request merged**